todo-list/src/App.vue

```diff
@@ -1,93 +1,86 @@
 <template>
   <div id="app" class="wrapper">
     <h1 class="title">TODO LIST</h1>
-    <div class="input__container">
-      <v-text-field
-        label="What to do?"
-        v-model="newTodo"
-        :rules="rules"
-        append-icon="mdi-plus"
-        @click:append="addTodo"
-      ></v-text-field>
-      <p>Totally {{ filteredItemsCount }} items found.</p>
-    </div>
+    <p>
+      <v-select
+        :items="[
+          { text: $t('lang.en'), value: 'en' },
+          { text: $t('lang.ru'), value: 'ru' },
+        ]"
+        @input="onChangeLocale"
+        v-model="locale"
```

Comment:
Do not mix v-model with explicit input event, better put watcher to `locale` prop and execute changes.